qiskit/pulse/schedule.py

```diff
@@ -926,65 +478,1112 @@ def draw(self,
+
+        Args:
+            start_time: Time to insert the schedule.
+            schedule: Schedule to insert.
+            name: Name of the new schedule. Defaults to the name of self.
+            inplace: Perform operation inplace on this schedule. Otherwise
+                return a new ``Schedule``.
+        """
+        pass
+
+    def append(self, schedule: BlockComponent,
+               name: Optional[str] = None,
+               inplace: bool = True) -> 'ScheduleBlock':
+        """Return a new schedule block with ``block`` appended to the context block.
+        The execution time is automatically assigned when the block is converted into schedule.
+
+        Args:
+            schedule: ScheduleBlock to be appended.
+            name: Name of the new ``Schedule``. Defaults to name of ``self``.
+            inplace: Perform operation inplace on this schedule. Otherwise
+                return a new ``Schedule``.
+
+        Returns:
+            Schedule block with appended schedule.
+
+        Raises:
+            PulseError: When invalid schedule type is specified.
+        """
+        if not isinstance(schedule, (ScheduleBlock, Instruction)):
+            raise PulseError(f'Appended `schedule` {type(schedule)} is invalid type. '
+                             'Only `Instruction` and `ScheduleBlock` can be accepted.')
+
+        if not inplace:
+            new_block = ScheduleBlock(
+                *self.instructions,
+                name=name or self.name,
+                metadata=self.metadata.copy(),
+                transform_policy=self.transform_policy,
+                **self.transform_opts
+            )
+            new_block.append(schedule, inplace=True)
+
+            return new_block
+        else:
+            self._blocks.append(schedule)
+            self._update_parameter_table(schedule)
+
+            return self
+
+    @_method_not_supported
+    def filter(self, *filter_funcs: List[Callable],
+               channels: Optional[Iterable[Channel]] = None,
+               instruction_types: Union[Iterable[abc.ABCMeta], abc.ABCMeta] = None,
+               time_ranges: Optional[Iterable[Tuple[int, int]]] = None,
+               intervals: Optional[Iterable[Interval]] = None,
+               check_subroutine: bool = True):
+        """Return a new ``Schedule`` with only the instructions from this ``ScheduleBlock``
+        which pass though the provided filters; i.e. an instruction will be retained iff
+        every function in ``filter_funcs`` returns ``True``, the instruction occurs on
+        a channel type contained in ``channels``, the instruction type is contained
+        in ``instruction_types``, and the period over which the instruction operates
+        is *fully* contained in one specified in ``time_ranges`` or ``intervals``.
+
+        If no arguments are provided, ``self`` is returned.
+
+        .. note:: This method is currently not supported.
```

Comment:
Here, I would say that support will be added soon. Might be nice if the warnings for methods that aren't yet supported (filter, exclude) is different from those that we are actually deprecating (insert, shift?)